Changelog

```diff
@@ -1,4 +1,83 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.5.0
+    - FEATURE: Add "Auto resize columns" functionality (Chocobo1)
+    - FEATURE: Allow to use Category paths in "Manual" mode (glassez)
+    - FEATURE: Allow to disable Automatic mode when "temp" path changed (glassez)
+    - FEATURE: Add tuning options related to performance warnings (Chocobo1)
+    - FEATURE: Add right click menu for status filters (An0n)
+    - FEATURE: Allow setting the number of maximum active checking torrents (An0n)
+    - FEATURE: Add option to toggle filters sidebar (AbeniMatteo)
+    - FEATURE: Allow to set "working set limit" on non-Windows OS (Chocobo1)
+    - FEATURE: Add "Export .torrent" action (Chocobo1)
+    - FEATURE: Add keyboard navigation keys (itlezy)
+    - FEATURE: Allow to use POSIX-compliant disk IO type (Coda)
+    - FEATURE: Add "Filter files" field in new torrent dialog (thalieht)
+    - FEATURE: Implement new icon/color theme (Nowshed H. Imran)
+    - FEATURE: Add file name filter/blacklist (mxtsdev, thalieht)
+    - FEATURE: Add support for custom SMTP ports (Emil M George)
+    - FEATURE: Split the OS cache settings into Disk IO read/write modes (summer)
+    - FEATURE: When duplicate torrent is added set metadata to existing one (glassez)
+    - FEATURE: Show startup progress dialog (glassez)
```

Comment:
I wouldn't call it a separate feature. However, it's up to you.

Reply:
> I wouldn't call it a separate feature. However, it's up to you.

In fact, this is only a small part of "Improve startup" work, which for some reason is not mentioned at all in this Changelog, although I treat it as most valuable from all my contributions to version 4.5.

Reply:
For some reason I totally missed that. Was it spread over multiple PRs? In any case, I change this entry to read:
```
- FEATURE: Greatly improve startup time with many torrents (glassez)
```

Reply:
> Was it spread over multiple PRs?

Yes, but the main one was called "Improve Startup" ;)
#16840

Reply:
To tell you the truth, I probably misjudged the commit title as too generic and didn't give more thought to it.